src/assets/sass/protocol/components/_notification-bar.scss

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../includes/lib";
+
+.mzp-c-notification-bar {
+    @include text-display-xxs;
+    background-color: $color-gray-20;
+    border-radius: 4px;
+    border: 1px;
+    box-shadow: 1px 1px 1px rgba(27,31,35,.1);
+    font-weight: normal;
+    margin: $layout-xs auto 0;
+    max-width: $content-sm;
+    text-align: center;
+
+    &:after {
+        content: "";
+        display: block;
+        clear: both;
+    }
+
+    &.mzp-is-sticky {
+        left: 0;
+        margin-left: auto;
+        margin-right: auto;
+        position: fixed;
+        right: 0;
+        top: 0;
+        z-index: 2;
+
+        .mzp-c-notification-bar-button {
+            width: 20px;
+
+            @media #{$mq-sm}{
+                width: 42px;
+            }
+        }
+    }
+
+    .mzp-c-notification-bar-button {
+        background: transparent url("#{$image-path}/icons/ui/close-black.svg") center center no-repeat;
+        border: none;
+        float: right;
+        height: 20px;
+        padding: 10px;
+
+        &:hover,
+        &:focus {
+            cursor: pointer;
+        }
+
+        @media #{$mq-sm} {
+            @include image-replaced;
```

Comment:
On breakpoints smaller than this I can see the close button text overflowing the bar, because this only applies to the `mq-sm` breakpoint and above.

The close button at this screen size also seems to be pretty small, which might be difficult as a touch target. I'd suggest maybe making some of these properties the default style?

![image](https://user-images.githubusercontent.com/400117/62476682-67c29800-b79f-11e9-8447-d50be09cfaad.png)